src/plugins/plot/chart/MctChart.vue

```diff
@@ -278,7 +278,7 @@ export default {
             // Have to throw away the old canvas elements and replace with new
             // canvas elements in order to get new drawing contexts.
             const div = document.createElement('div');
-            div.innerHTML = this.TEMPLATE;
+            div.innerHTML = this.canvasTemplate + this.canvasTemplate;
```

Comment:
Do we _really_ want to append this to itself? I don't understand this code.

Reply:
This is a case where describing the change per the PR template might provide some clarity.

Reply:
We're creating something like `<canvas></canvas><canvas></canvas>`.
The first canvas element is for the overlay and 2nd is the main canvas. Although, I think the 2D draw API doesn't need an overlay, so we could get rid of it. I think this was done just to be consistent between APIs.